src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Contains the name of the tag removed/added and the name of the person.
+ */
+
+public class Tagging {
+
+	private String addOrDel;
+	private Person person;
+	private Tag tag;
+	
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.